tests/integration/targets/nxos_static_routes/tests/cli/overridden.yml

```diff
@@ -0,0 +1,58 @@
+---
+- debug:
+    msg: "Start nxos_static_routes overridden tests connection={{ ansible_connection }}"
+
```

Comment:
Please a remove_config call here, just to make sure we're not working on top of a spill over config.